packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -930,6 +930,14 @@ export const Table = <
                   {headerGroup.headers.map((column, index) => {
                     const { onClick, ...restSortProps } =
                       column.getSortByToggleProps();
+
+                    const columnHasExpanders =
+                      hasAnySubRows &&
+                      index ===
+                        headerGroup.headers.findIndex(
+                          (c) => c.id !== SELECTION_CELL_ID,
+                        );
```

Comment:
Is this selecting the first non-selection column's header? Maybe separating this to a new variable with a description variable name (e.g. `isFirstNonSelectionColumn`) or at least a comment explanation here would make it more understandable.

Reply:
Added comment [`5027473`](https://github.com/iTwin/iTwinUI/pull/1796/commits/5027473e80360d34ce509ed1adb1ef1adeb39aa3)